src/components/Blog/PostListItem.tsx

```diff
@@ -71,16 +72,16 @@ export const PostListItem: FunctionComponent<PostIndexItemProps> = ({
                         )}
                         <div className="text-center text-sm-left">
                             <Link href={`/${blogType}/${slugPath}`} passHref={true}>
-                                {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                <a className="font-weight-bold">Read more</a>
+                                <a className="font-weight-bold" href="#none" title="Read more">
```

Comment:
Should we treat every anchor tag as a CTA? Like here should the image & author link be a CTA too?
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/59381432/179613493-3e999324-90ca-496d-8833-9c9ba5a2c0e2.png">

Reply:
That's a good question and something I pondered about and am battling with in my mind haha! 😅 Technically we could add tracking to literally every link element since we now have `text` as a cta-style option for example. If you'd like to run through every link, feel free! It doesn't hurt; the more tracking the better!

Reply:
Alright, I added `image` as a styling option too and added tracking to every link in the repo in [f0cae8f](https://github.com/sourcegraph/about/pull/5574/commits/f0cae8ff2624925a9b7cd0a9f26bb469c070a46e) & [cd85c84](https://github.com/sourcegraph/about/pull/5574/commits/cd85c8468599f9727eb7f20c44d5ec79a1e7a650). Everything is covered now!